Cloudinary-pwa-kit/packages/pwa-cloudinary/overrides/app/components/footer/index.jsx

```diff
@@ -0,0 +1,290 @@
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+    Box,
+    Text,
+    Divider,
+    SimpleGrid,
+    useMultiStyleConfig,
+    Select as ChakraSelect,
+    Heading,
+    Input,
+    InputGroup,
+    InputRightElement,
+    createStylesContext,
+    Button,
+    FormControl
+} from '@chakra-ui/react'
+import { useIntl } from 'react-intl'
+
+import LinksList from '@salesforce/retail-react-app/app/components/links-list'
+import SocialIcons from '@salesforce/retail-react-app/app/components/social-icons'
+import { HideOnDesktop, HideOnMobile } from '@salesforce/retail-react-app/app/components/responsive'
+import { getPathWithLocale } from '@salesforce/retail-react-app/app/utils/url'
+import LocaleText from '@salesforce/retail-react-app/app/components/locale-text'
+import useMultiSite from '@salesforce/retail-react-app/app/hooks/use-multi-site'
+import styled from '@emotion/styled'
+
+{/** Cloudinary Custom Code Starts */}
+import Helmet from 'react-helmet'
+import { cloudinary } from '../../../../config/default'
+{/** Cloudinary Custom Code Ends */}
+
+const [StylesProvider, useStyles] = createStylesContext('Footer')
+const Footer = ({ ...otherProps }) => {
+    const styles = useMultiStyleConfig('Footer')
+    const intl = useIntl()
+    const [locale, setLocale] = useState(intl.locale)
+    const { site, buildUrl } = useMultiSite()
+    const { l10n } = site
+    const supportedLocaleIds = l10n?.supportedLocales.map((locale) => locale.id)
+    const showLocaleSelector = supportedLocaleIds?.length > 1
+
+    // NOTE: this is a workaround to fix hydration error, by making sure that the `option.selected` property is set.
+    // For some reason, adding some styles prop (to the option element) prevented `selected` from being set.
+    // So now we add the styling to the parent element instead.
+    const Select = styled(ChakraSelect)({
+        // Targeting the child element
+        option: styles.localeDropdownOption
+    })
+
+    return (
+        <Box as="footer" {...styles.container} {...otherProps}>
+
+            {/** Cloudinary Custom Code Starts */}
+            <Helmet>
+                <script src={`https://unpkg.com/cloudinary-core@${cloudinary.versions.CLDCoreShrinkwrapJSURLVersion}/cloudinary-core-shrinkwrap.min.js`} />
+            </Helmet>
+            <Helmet>
+                <script src={`https://unpkg.com/cloudinary-video-player@${cloudinary.versions.cldVideoPlayerJsVersion}/dist/cld-video-player.min.js`} />
+            </Helmet>
+            <Helmet>
+                <link rel="stylesheet" href={`https://unpkg.com/cloudinary-video-player@${cloudinary.versions.cldVideoPlayerCssVersion}/dist/cld-video-player.min.css`} />
+            </Helmet>
+            <Helmet>
+                <script src={`https://product-gallery.cloudinary.com/all.js`} />
```

Comment:
We already have the gallery script in pwa-cloudinary/overrides/app/pages/product-detail/index.jsx with a conditional usage. Do we need it here as well

Reply:
yes we need it

Reply:
@asad-rafter ^